File: S3/BASE_DATOS/SoftProgRRHHController/src/pe/edu/pucp/softprog/rrhh/dao/AreaDAO.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package pe.edu.pucp.softprog.rrhh.dao;

import java.util.ArrayList;
import pe.edu.pucp.lp2soft.rrhh.model.Area;

/**
 *
 * @author Danielrp551
 */

public interface AreaDAO {
    int insertar(Area area);
    int modificar(Area area);
    int eliminar (int id_area);
    ArrayList<Area> listarTodas();
}

File: S5/SoftProgRRHHModel/SoftProgRRHHController/DAO/AreaDAO.cs
﻿using SoftProgRRHHModel;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftProgRRHHController.DAO
{
    public interface AreaDAO
    {
        int insertar(Area area);
        int modificar(Area area);
        int eliminar(int  idArea);

        BindingList<Area> listarTodas();
    }
}
